src/example/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Audience } from "@streamdal/snitch-protos/protos/common.js";
```

Comment:
helper example in the source itself...just for me so I can run examples against source changes and not against the dep.